pmd-java/src/main/java/net/sourceforge/pmd/lang/java/ast/ASTStringLiteral.java

```diff
@@ -29,9 +29,11 @@ public final class ASTStringLiteral extends AbstractLiteral implements ASTLitera
     }
 
 
-    // todo deprecate this
-    // it's ambiguous whether it returns getOriginalText or getTranslatedText
+    /**
+     * @deprecated use {{@link #getText()}}
+     */
     @Override
+    @Deprecated
     public String getImage() {
```

Comment:
Would that be something, we keep deprecated in PMD 7? Or do we remove it?

Reply:
I'm not sure... If we remove it then the image will not be available in XPath rules (although there is `@ConstValue` which is slightly different). OTOH I want to avoid java rules using the image instead of the text if possible...